src/components/AssociatedStudiesTable.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { OtTable, commaSeparate } from 'ot-ui';
+
+import LocusLink from './LocusLink';
+
+const tableColumns = (geneId, chromosome, position) => [
+  {
+    id: 'studyId',
+    label: 'Study ID',
+    renderCell: rowData => (
+      <Link to={`/study/${rowData.studyId}`}>{rowData.studyId}</Link>
+    ),
+  },
+  {
+    id: 'traitReported',
+    label: 'Trait',
+  },
+  {
+    id: 'pmid',
+    label: 'PMID',
+    renderCell: rowData => (
+      <a
+        href={`http://europepmc.org/abstract/med/${rowData.pmid}`}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        {rowData.pmid}
+      </a>
+    ),
+  },
+  {
+    id: 'pubAuthor',
+    label: 'Author (Year)',
+    renderCell: rowData =>
+      `${rowData.pubAuthor} et al (${new Date(rowData.pubDate).getFullYear()})`,
```

Comment:
Since the API is in charge of providing the correct `pubAuthor`, we can remove `et al` in the front end code.